mods/default/locale/default.de.tr

```diff
@@ -0,0 +1,205 @@
+# textdomain: default
+Locked Chest=Abgeschlossene Truhe
+Locked Chest (owned by @1)=Abgeschlossene Truhe (Eigentümer: @1)
+You do not own this chest.=Ihnen gehört diese Truhe nicht
+a locked chest=eine abgeschlossene Truhe
+Chest=Truhe
+Stick=Stock
+Paper=Papier
+"@1" by @2=„@1“ von @2
+Book=Buch
+Book with Text=Buch mit Text
+Skeleton Key=Generalschlüssel
```

Comment:
should match the original in naming, not just "intent"
so e.g. `Skelettschlüssel`